queries/java/highlights.scm

```diff
@@ -2,20 +2,29 @@
 
 ; Methods
 
+[
 (method_declaration
-  name: (identifier) @method)
+ name: (identifier))
 (method_invocation
-  name: (identifier) @method)
+ name: (identifier))
+] @method
```

Comment:
With this kind of this you are going to capture to whole `method_delcaration` group, which for sure is not right, you will have to keep it capturing the `(identifier)` node alone.

Reply:
> you will have to keep it capturing the (identifier) node alone

That would capture all identifiers instead of method identefiers only, no?